docs/index.rst

```diff
@@ -45,6 +45,74 @@ the services follow these instructions:
 This installs the executables ``pathfinding-service``, ``monitoring-service``
 and ``request-collector`` into the virtual environment.
 
+Using MS and PFS in raiden
+--------------------------
+We need to install raiden from the latest development branch.
+
+.. code:: bash
+
+    git clone https://github.com/raiden-network/raiden.git
+    cd raiden
+    git checkout development
+    make install-dev
+
+Raiden client needs to be started with the raiden-services flags like
+
+* --enable-monitoring
+* --routing-mode [pfs|local|private]
+
+Doing ``raiden --help`` and going to Raiden Services options would give more
+information.
+
+Payments related to Services
```

Comment:
What target group is this written for? I'd say users don't care about the contracts...

Reply:
Yes users exclusively of the MS and PFS services.